### ISSUE TITLE
Ensure interactive controls are not nested as they are not always announced by screen readers or can cause focus problems for assistive technologies

### ISSUE THREAD

**Issue by gouthamp2428** [tinymce/tinymce#10994]
## 📝 Provide a description of the new feature or improvement
*Describe your idea for the new feature or improvement*
All interactive controls should be independent and not nested. Screen readers should announce each control correctly, and keyboard focus should behave predictably, allowing users to interact with all controls efficiently.

### 🫶 What is the motivation?
*Pitch why this feature or improvement should be implemented.*
Improves accessibility for user with different abilities

### 🔗 What is the consequence of not having this feature?
*This helps understand the underlying pain-points*
Differently abled users may choose different web experience

### 🚦 How important would you rate the requested feature or improvement?
*Choose from nice-to-have, important or critical.*
important
---
Screenshot 

<img width="1098" height="285" alt="Image" src="https://github.com/user-attachments/assets/ccc8c21e-a13c-4714-9af6-57b6fde7a2f5" />

If you'd like to see this implemented sooner, add a 👍 reaction to this post.
